test/image/mocks/zz-shapes_layer_below_traces.json

```diff
@@ -0,0 +1,126 @@
+{
+  "data": [
+    {
+      "name": "scatter",
+      "x": [
+        "A",
+        "B",
+        "C",
+        "D",
+        2000.0
+      ],
+      "y": [
+        100,
+        200,
+        150,
+        190,
+        50
+      ],
+      "type": "scatter"
+    },
+    {
+      "name": "bar",
+      "x": [
+        "A",
+        "B",
+        "C",
+        "D",
+        2000.0
+      ],
+      "y": [
+        50,
+        70,
+        90,
+        35,
+        55
+      ],
+      "type": "bar"
+    }
+  ],
+  "layout": {
+    "shapes": [
+      {
+        "name": "dashdot line",
+        "fillcolor": "rgba(179,179,179,0.6)",
+        "layer": "above",
+        "line": {
+          "color": "rgba(66,127,109,0.6)",
+          "dash": "dashdot"
+        },
+        "type": "line",
+        "x0": 0.75,
+        "x1": 0.75,
+        "xref": "x",
+        "y0": 0.0,
+        "y1": 1.0,
+        "yref": "paper"
+      },
+      {
+        "name": "horizontal line",
+        "fillcolor": "rgba(179,179,179,1)",
+        "layer": "above",
+        "line": {
+          "color": "rgba(0,0,0,1)"
+        },
+        "type": "line",
+        "x0": 0.0,
+        "x1": 1.0,
+        "xref": "paper",
+        "y0": 120.0,
+        "y1": 120.0,
+        "yref": "y"
+      },
+      {
+        "name": "blue rect",
+        "fillcolor": "rgba(108,173,225,0.8)",
```

Comment:
The transparency on this rectangle makes it hard to tell whether it's above or below the gridline. Recommend we stick to fully opaque shapes for this test.

Also -- add text to one of these shapes to test that it's rendered correctly on the `between` layer?

Reply:
done and done!